server/protocols/webadmin-cli/README.md

```diff
@@ -0,0 +1,128 @@
+# A WebAdmin based CLI for Apache James
+
+## Development
+
+Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface. It also aims at providing a more modern and intuitive interface.
+
+## Run the script
+
+## Syntax
+
+General syntax to run the script
+
+```   
+$ ./james_cli [OPTION] ENTITY ACTION {ARGUMENT}
+```
+
+where
+
+    OPTION: optional parameter when running the script,
+  
+    ENTITY: represents the route performing actions,
```

Comment:
represents the entity to perform action on

Reply:
i fixed it.